lib/jsonlint/rake_task.rb

```diff
@@ -24,7 +30,14 @@ def define_task
       task(name) do
         puts 'Running JsonLint...'
 
-        files_to_check = Rake::FileList.new(paths)
+        YamlLint.logger.level = Logger::DEBUG if debug
+
+        files_to_check_raw = Rake::FileList.new(paths)
+        files_to_exclude = Rake::FileList.new(exclude_paths)
+        files_to_check = files_to_check_raw - files_to_exclude
```

Comment:
Any reason not to use `Rake::FileList#exclude` to incrementally build up the `FileList` to iterate over? I'm a bit worried that this approach will have subtle behavioral or performance differences.